src/Components/AddForm.js

```diff
@@ -1,9 +1,13 @@
 import PropTypes from 'prop-types';
 import React, { Component } from 'react';
-import './AddForm.css';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
 import shortid from 'shortid';
+// action
+import { addItem } from '../actions';
+import './AddForm.css';
 
-class AddForm extends Component {
+export class AddForm extends Component {
   //  the method doesn’t use this, so make it a static method
   static showModal() {
```

Comment:
When you're using `React`, avoid to manipulate the real `DOM`.
For this case you can use `state` to handle the modal open or close
You can write something like following
```js
this.state = {
  isOpen: true || false
}

// In render()
const { isOpen } = this.state
<div className=`modal fade ${isOpen ? 'show' : 'hide'}` id="myModal">
```